CMakeLists.txt

```diff
@@ -1,9 +1,7 @@
 cmake_minimum_required(VERSION 3.5)
 project(ndt_omp)
 
-# should use march=native ?
-add_definitions(-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2)
-set(CMAKE_CXX_FLAGS "-std=c++14 -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2")
+add_compile_options(-march=native)
```

Comment:
Will removing `-msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2` not affect the performance?
Could you link any references around these options?

Also, are there any reasons for removing `-std=c++14`?

Reply:
> Will removing -msse -msse2 -msse3 -msse4 -msse4.1 -msse4.2 not affect the performance?

Yes, @harihitode san evaluated the performance.
https://tier4.atlassian.net/browse/T4PB-12676?focusedCommentId=69870

Here is the reference.
http://eigen.tuxfamily.org/index.php?title=FAQ#How_can_I_enable_vectorization.3F

> Also, are there any reasons for removing -std=c++14?

Ah, I guess we need to keep this option. It's better to use `set(CMAKE_CXX_STANDARD 14)` instead.

Reply:
> Yes, @harihitode san evaluated the performance.
https://tier4.atlassian.net/browse/T4PB-12676?focusedCommentId=69870

Since it's an internal link, we should copy the result here so that everyone can see it.

> Ah, I guess we need to keep this option. It's better to use set(CMAKE_CXX_STANDARD 14) instead.

Yes, but making it another PR is better.

Reply:
> Yes, but making it another PR is better.

OK. Then I restore c++14 option.
3b3b92a

Reply:
> Since it's an internal link, we should copy the result here so that everyone can see it.

![image](https://user-images.githubusercontent.com/19993104/149280282-2e7fe0dd-f730-4577-82d6-a222aa8c4b94.png)

Reply:
Thank you, but you should write the condition of the experiment.

Reply:
@kenji-miyake @KeisukeShima The result is obtained on Intel machine using rosbag playing simulation.

CPU: i9-9900 supporting AVX256 (256 bit SIMD)
We play ROSBAG as instructed in https://tier4.github.io/autoware.proj/tree/main/tutorial/QuickStart/ .

Red line is our proposal which uses 256 bit SIMD in the execution binary. Current default is the blue line, 128 bit SIMD.
Because this program itself is not tuned to the 256 bit SIMD, we have no performance-up either performance degradation.

Reply:
@yukkysaito Thanks for your comment. I think this change has no effect on its performance, but I have evaluated it only on rosbag-simulation.